.github/workflows/ci.yml

```diff
@@ -228,9 +228,21 @@ jobs:
       - run: nix flake check
         continue-on-error: true
 
+  check-links:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@f87f0a62993c2647717456af92593666acb3a500 # lychee v0.16.1 seems to work well for our case at the moment
```

Comment:
Could you comment with what exactly is broken and link to issue? Also is there maybe a working tag rather than a commit hash?